site/content/contribute/getting-started/core-committers.md

```diff
@@ -56,9 +56,9 @@ Below is the list of core committers working on Mattermost:
 - **<a name="hyeseong.kim">Hyeseong Kim</a>**
     - @cometkim on [community.mattermost.com](https://community.mattermost.com/core/messages/@cometkim) and [@cometkim](https://github.com/cometkim) on GitHub
     - Dev areas: Web App, React, Redux, REST API
-- **<a name="hanzei">Hanzei</a>**
+- **<a name="ben.schumacher">Ben Schumacher</a>**
     - @hanzei on [community.mattermost.com](https://community.mattermost.com/core/messages/@hanzei) and [@hanzei](https://github.com/hanzei) on GitHub
-    - Dev areas: Plugins, Integrations, Security
+    - Dev areas: Developer Toolkit, Plugins, Community
```

Comment:
We decides the go with the term `Toolkit` instead of `Developers Toolkit` for the discussion channel while ago. What do you think about also applying this decision to this page and holistically change it?

Reply:
Works for me, and pushed!